src/components/MediatedRequestsActivities/components/RequesterInformation/RequesterInformation.js

```diff
@@ -277,41 +282,39 @@ class RequesterInformation extends Component {
                             onBlur={this.handleBlur(input)}
                             onKeyDown={this.onKeyDown}
                             disabled={!isEditPermission}
+                            className={css.requesterBarcodeField}
                           />
                         );
                       }}
                     </Field>
                   );
                 }}
               </FormattedMessage>
-              {
-                isEditPermission &&
-                  <Pluggable
-                    aria-haspopup="true"
-                    type="find-user"
-                    searchLabel={<FormattedMessage id="ui-requests-mediated.form.requester.lookupLabel" />}
-                    searchButtonStyle="link"
-                    dataKey="users"
-                    selectUser={this.onSelectUser}
-                    visibleColumns={VISIBLE_COLUMNS}
-                    columnMapping={COLUMN_MAPPING}
-                    disableRecordCreation
-                    marginTop0
-                  />
-              }
-            </Col>
-            <Col xs={3}>
               <Button
-                buttonStyle="primary noRadius"
+                buttonStyle="default"
                 buttonClass={enterButtonClass}
-                fullWidth
                 onClick={this.handleClick}
                 disabled={isEnterButtonDisabled}
               >
                 <FormattedMessage id="ui-requests-mediated.form.enterButton" />
               </Button>
             </Col>
           </Row>
+          {
+            isEditPermission &&
+              <Pluggable
+                aria-haspopup="true"
+                type="find-user"
+                searchLabel={<FormattedMessage id="ui-requests-mediated.form.requester.lookupLabel" />}
+                searchButtonStyle="link"
+                dataKey="users"
+                selectUser={this.onSelectUser}
```

Comment:
Should this be bound or should the arrow function be used here to avoid losing the context?

Reply:
`onSelectUser` is an arrow function, it should not lose context.
Moreover, I did not change already existing logic, I just moved `Pluggable` from one place to another. 
In other words it will work as previously.